RecoEgamma/EgammaIsolationAlgos/python/gamIsoDeposits_cff.py

```diff
@@ -6,11 +6,11 @@
 from RecoEgamma.EgammaIsolationAlgos.gamIsoDepositEcalFromHits_cff import *
 from RecoEgamma.EgammaIsolationAlgos.gamIsoDepositHcalFromTowers_cff import *
 
-gamIsoDeposits = cms.Sequence(
-    gamIsoDepositTk + 
-    gamIsoDepositEcalFromHits + 
-    gamIsoDepositHcalFromTowers +
-    gamIsoDepositHcalDepth1FromTowers +
+gamIsoDepositsTask = cms.Task(
+    gamIsoDepositTk , 
+    gamIsoDepositEcalFromHits , 
+    gamIsoDepositHcalFromTowers ,
+    gamIsoDepositHcalDepth1FromTowers ,
     gamIsoDepositHcalDepth2FromTowers
 )
-
+gamIsoDeposits = cms.Sequence(gamIsoDepositsTask)
```

Comment:
This is only used in ```RecoEgamma/EgammaIsolationAlgos/python/gamIsolationSequence_cff.py```: maybe one could avoid defining explicitly the sequence here